Sources/ShowTouches/UIWindow+ShowTouches.swift

```diff
@@ -32,17 +34,27 @@ extension UIWindow {
 			}
 		}
 	}
-
-	static var didStart = false
-	@objc public static func startShowingTouches() {
-		guard !didStart else { return }
-		didStart = true
-
+	
+	public static func configure(_ config: ShowTouchesConfig) {
+		Self.config = config
+	}
+	
+	@objc public static func showTouches(_ show: Bool = true) {
 		guard let originalMethod = class_getInstanceMethod(UIWindow.self, #selector(sendEvent(_:))),
 		      let newMethod = class_getInstanceMethod(UIWindow.self, #selector(showTouches_sendEvent(_:)))
 		else {
 			return
 		}
-		method_exchangeImplementations(originalMethod, newMethod)
+		
+		if show {
+			method_exchangeImplementations(originalMethod, newMethod)
+		} else {
+			method_exchangeImplementations(newMethod, originalMethod)
+		}
```

Comment:
The 2 branches do the same thing, so if we call it with `show: true` twice then it will be turned back off.

Reply:
🙈
Good catch!
Should be fixed now.